src/azure-cli/azure/cli/command_modules/acr/tests/latest/recordings/test_check_name_availability.yaml

```diff
@@ -17,8 +17,8 @@ interactions:
       ParameterSetName:
       - -n
       User-Agent:
-      - python/3.7.9 (Windows-10-10.0.19041-SP0) msrest/0.6.18 msrest_azure/0.6.3
-        azure-mgmt-containerregistry/3.0.0rc16 Azure-SDK-For-Python AZURECLI/2.16.0
+      - python/3.8.10 (Windows-10-10.0.19041-SP0) msrest/0.6.21 msrest_azure/0.6.3
```

Comment:
is checking in these new recordings necessary? If not, it's okay, but also consider removing unnecessary check-ins to keep PR concise.